validator/src/cli.rs

```diff
@@ -1381,6 +1381,35 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .possible_values(BlockProductionMethod::cli_names())
                 .help(BlockProductionMethod::cli_message())
         )
+        .arg(
+            Arg::with_name("wen_restart")
```

Comment:
Took a quick look at this because of https://github.com/solana-labs/solana/issues/33633. I believe (but could be mistaken) that this feature is not yet production ready. If that is correct, i think we should add this
```
.hidden(hidden_unless_forced())
```
with the idea being to keep the in-development features hidden from `solana-validator --help` until they're ready